core/lib/Cardano/Address/Style/Icarus.hs

```diff
@@ -199,7 +219,7 @@ instance Internal.GenMasterKey Icarus where
 instance Internal.HardDerivation Icarus where
     type AccountIndexDerivationType Icarus = 'Hardened
     type AddressIndexDerivationType Icarus = 'Soft
-    type WithAccountStyle Icarus = AccountingStyle
+    type WithAccountStyle Icarus = Role
```

Comment:
Perhaps also rename that to `WithRole` ?

Reply:
done in https://github.com/input-output-hk/cardano-addresses/pull/75/commits/e601936c1196486dcee8723fe7be9a20224ce6c7

Reply:
@paweljakubas I think what @KtorZ meant is rename the type synonym family `WithAccountStyle` to `WithRole` but keep the famliy instance type as `Role`.

Reply:
indeed! I fixed this in https://github.com/input-output-hk/cardano-addresses/pull/75/commits/f572fb0902eaa9d4d0f1713b66ea983bc627cbc8 (and squashed with previous commit)